src/lib/divider/divider.spec.ts

```diff
@@ -0,0 +1,99 @@
+import {async, TestBed} from '@angular/core/testing';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {MatDividerModule} from './divider-module';
+
+
+describe('MatDivider', () => {
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [MatDividerModule],
+      declarations: [
+          MatDividerBase,
+          MatDividerVertical,
+          MatDividerInset,
+          MatDividerVerticalInset,
+          MatDividerWithCssClass,
+      ],
+    });
+
+    TestBed.compileComponents();
+  }));
+
+  it('should not apply any additional class to a list without lines', () => {
+    let fixture = TestBed.createComponent(MatDividerBase);
```

Comment:
You can use `const` here and where possible.

Reply:
This was based on every other spec file in the Material lib, which uses `let` instead of `const`. That doesn't make it better, so I switched it to `const`.